core/src/server/web_impl/README.md

```diff
@@ -681,7 +681,7 @@ Updates the index type and nlist of a collection.
 | Parameter    | Description                                                                                                                                                                                              | Required? |
 | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- |
 | `index_type` | The type of indexing method to query the collection. Please refer to [Index Types](https://www.milvus.io/docs/reference/index.md) for detailed introduction of supported indexes. The default is "FLAT". | No        |
-| `params`     | The extra params of indexing method to query the collection. Please refer to - [Index params](#index_param) for detailed introduction of supported indexes.                                              | No        |
```

Comment:
Please refer to [Index and Search Parameters](#Index-and-Search-Parameters)

Reply:
In this way, the links will work.

Reply:
solved.